### PR TITLE
feat: add the exclusive features html

### PR DESCRIPTION
**Pull request by GusBaamonde**
add navigation for exclusive features
add the css for the checkboxs
display change to improve validation alignment

caracteristicas_exclusivas

caracteristicas_exclusivas_2

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/modules/_forms.scss [hunk 1]
```diff
@@ -321,7 +321,7 @@
     /* Messages and confirmations the forms */
     .dp-message,
     .wrapper-errors {
-      display: inline-block;
+      display: block;
```
Comment (cbernat) [inline assets/scss/modules/_forms.scss:324]: Are you sure this change won´t break other validations? it seems widely used to me... 🤔

**Comment by cbernat** [comment]
image
🤔 
image
This is looking weird...

**Comment by GusBaamonde** [comment]
plan_agencias

**Comment by GusBaamonde** [comment]
caracteristicas_exclusivas2

**Comment by cbernat** [comment]
Wonderful!!! ❤️

**Pull request merged**